plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -268,6 +270,19 @@ public BigQueryConfig setMetadataCacheTtl(Duration metadataCacheTtl)
         return this;
     }
 
+    public boolean isLegacyMetadataListing()
+    {
+        return isLegacyMetadataListing;
+    }
+
+    @Config("bigquery.legacy-metadata-listing")
+    @ConfigDescription("Use BigQuery API to list metadata")
```

Comment:
Is legacy using BigQuery API or your new implementation?

Reply:
Updated the description.